### PR TITLE
Style for OMT 3.11 - boundary, water brunnel

### PR DESCRIPTION
**Pull request by eva-j**
- do not display parts of water bodies that are under surface (you can check in Paris: 15/48.85613/2.37204)
- do not display disputed boundary relations in zooms 0-4 (there is Natural Earth data source)

### CODE REVIEW

**Comment by pathmapper** [comment]
Hi @eva-j,

thanks for your PR.

I think the brunnel filter should also be applied to the waterway layers (`waterway_river` and `waterway_other`) because there is still water under the surface visible (Paris area you mentioned). What do you think?

grafik

Regarding the boundaries: 
I don't see any boundaries with a `claimed_by` field in the current tiles. Can you point me to boundaries where this is the case or is this PR for a future change in the tiles?
According to the [OMT changelog](https://github.com/openmaptiles/openmaptiles/releases/tag/v3.11), relations with `claimed_by` should be already included in OMT v3.11.

**Comment by eva-j** [comment]
Hi @pathmapper, thanks for having a look!

I've just added waterway_tunnel layer, hopefully it looks better.
Anyway, the source
```
    "openmaptiles": {
      "type": "vector",
      "url": "https://api.maptiler.com/tiles/v3/tiles.json?key={key}"
```
will be updated to v3.11 during the weekend, so the changes will become visible.

File: style.json [hunk 1]
```diff
@@ -120,12 +120,27 @@
       "filter": ["==", "class", "school"],
       "paint": {"fill-color": "rgb(236,238,204)"}
     },
+    {
+      "id": "waterway_tunnel",
+      "type": "line",
+      "source": "openmaptiles",
+      "source-layer": "waterway",
+      "filter": ["all", ["==", "brunnel", "tunnel"]],
+      "layout": {"visibility": "visible"},
```
Comment (pathmapper) [inline style.json]:
```suggestion

```

Could you please remove this line, it's not needed and we recently cleaned up the style in https://github.com/maputnik/osm-liberty/pull/61 :smile:

**Comment by pathmapper** [comment]
@eva-j thanks for adding the `waterway_tunnel` layer. Makes sense because we are showing currently also the names for subsurface waterways.

~~The issue I meant is related to this riverbank 
https://www.openstreetmap.org/way/236735249#map=18/48.85285/2.36905~~

~~I think there is currently no option to hide it, therefore I opened:
https://github.com/openmaptiles/openmaptiles/issues/764~~

Left one comment regarding a small change, otherwise this PR is good to merge.

**Comment by pathmapper** [comment]
> updated to v3.11 during the weekend, so the changes will become visible.

This fixed the riverbank issue.

File: style.json (hunk 1 above)
Comment (eva-j) [reply style.json]: done ;)

File: style.json [hunk 2]
```diff
@@ -1241,10 +1256,25 @@
       }
     },
     {
-      "id": "boundary_2",
+      "id": "boundary_2_z0-4",
+      "type": "line",
+      "source": "openmaptiles",
+      "source-layer": "boundary",
+      "maxzoom": 5,
```
Comment (boldtrn) [inline style.json:1263]: Is maxzoom: 5 correct here?

Comment (eva-j) [reply style.json:1263]: Yes, there is different source of boundary data in z0-4 (Natural Earth) and then starting z5 (OSM).

Comment (boldtrn) [reply style.json:1263]: Ah I see, thanks for the explanation.

**Pull request merged**